extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -143,7 +143,9 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       closeButtonEl.setAttribute('aria-label', localizedCloseString);
     }
 
-    if (this.element.hasAttribute('data-title')) {
+    if (this.element.hasAttribute('title')) {
+      titleEl.textContent = this.element.getAttribute('title');
+    } else if (this.element.hasAttribute('data-title')) {
```

Comment:
Nit: can we factorize?

e.g. `const title = this.element.getAttribute('title') || getAttribute('data-title')`